evm/vm/computation.py

```diff
@@ -228,7 +229,8 @@ def extend_memory(self, start_position: int, size: int) -> None:
         before_cost = memory_gas_cost(before_size)
         after_cost = memory_gas_cost(after_size)
 
-        self.logger.debug(
+        self.logger.log(
+            LOG_TRACE,
```

Comment:
Same here, we are using `logger.trace(...)` in that same file already. Can't we use that?